README.md

```diff
@@ -89,10 +153,11 @@ glasswall.config.logging.console.setLevel(logging.DEBUG)
 glasswall.config.logging.log.setLevel(logging.DEBUG)
 ```
 
-</details>
+---
 
-<details>
-<summary>Content management policies</summary>
+### Content management policies
+
+Documentation about content management policies can be found here: https://docs.glasswall.com/docs/glasswall-engine-content-management
```

Comment:
Documentation about content management policies can be found on the [Policy Management](https://docs.glasswall.com/docs/embedded-engine-policy-management) page.